src/portal.ts

```diff
@@ -71,6 +72,42 @@ class DeployCommand extends Command {
   }
 }
 
+class SyncCommand extends Command {
+  public static usage = Command.Usage({
+    description: 'Sync changes made locally under the given workspace upstream with the upstream workspace.',
+    details: `
+    This command will sync local templates from given workspace folder to the same workspace upstream.\n
+    If \`-W,--watch\` option is given after all the local templates are deployed the deploy will stay running and push any new changes on the filesystem in the workspace.\n
```

Comment:
This command will sync local templates from given workspace folder to the same workspace upstream.\n

    templates and other files, too, yeah?